module_utils/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
remove this file (at .vscode to .gitignore)

Reply:
okay, will remove it through .gitignore